flink/src/main/resources/interpreter-setting.json

```diff
@@ -16,6 +16,9 @@
         "defaultValue": "6123",
         "description": "port of running JobManager."
       }
+    },
+    "editor": {
+      "mode": "scala"
```

Comment:
In my personal opinion,
"Mode" is not a bad name. But better "language" right.
mode stands for 'launguage mode' of ui-ace.
However, Interpreter shall not directly related to the ui-ace.
In addition, its value is "kind of language."
I believe that the future of Zeppelin may not use the ace editor.
If so, the name "mode" can cause confusion.
What do you think?